_posts/2022-08-15-requiring-mfa-on-popular-gems.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Requiring MFA on popular gem maintainers
+layout: post
+author: Jenny Shen
+author_email: jenny.shen@shopify.com
+---
+<p align="center">
+    <img src="/images/gem-with-mfa-flag-dropshadow.png" alt="Doodle of a RubyGem wearing an MFA hat, holding a flag with a checkmark" width="300"/>
+</p>
+
+Two months ago, we outlined our [commitment](https://blog.rubygems.org/2022/06/13/making-packages-more-secure.html) to making Ruby’s supply chain more secure. To combat account takeovers — the second most common software supply chain attack —  we announced a policy to require MFA on at least the top-100 RubyGems packages.
+
+Today (August 15th, 2022), we will begin to enforce MFA on owners of gems with over 180 million total downloads. Users in this category who do not have MFA enabled on the `UI and API` or `UI and gem signin` level will not be able to access profile pages on the web, perform [privileged actions](https://guides.rubygems.org/mfa-requirement-opt-in/#privileged-operations) (i.e. push and yank gems, or add and remove gem owners) or sign in on the command line until they [configure MFA](https://guides.rubygems.org/setting-up-multifactor-authentication/).
+
+Maintainers of gems that surpass 165 million total downloads will continue to receive recommendation reminders on the UI and CLI until the gem reaches 180 million total downloads. At that point, MFA will be required.
+
+This policy would bring us in line with the policies made by other package ecosystems. We have plans to increase MFA adoption on RubyGems, which will be announced in the near future. In addition, we are also currently working on adding support for [WebAuthn](https://webauthn.guide/). Maintainers would be able to use hardware tokens, biometric keys, and other WebAuthn-supported devices as their multi-factor device of choice.
```

Comment:
> We have plans to increase MFA adoption on RubyGems, which will be announced in the near future.

I plan to go back to https://github.com/rubygems/rfcs/issues/42 in the coming days. Unsure if we should link it in the blog post (I'm guessing a decision would not be reached in time for this release so perhaps we should link them to the most recent GH discussion about it whether that is the issue or a formal RFC that will be opened).

Reply:
Given the tepid engement (on github) of last time I doubt there would be as many enthusist this time either. I would link the issue for the sake of transparency. It's more about giving everyone a chance to speak and sway decisions as oppose to telling that this is what is happening.

Reply:
Makes sense! I added a line to link to the issue
`If you have ideas on how future rollouts should be approached, join this [discussion](https://github.com/rubygems/rfcs/issues/42) in our RFC repository!`